### PR TITLE
fix: FTP and SFTP connectors

### PR DESCRIPTION
**Pull request by sunilmeddr**
Fixes: #1185
Fixes: #1186

### CODE REVIEW

File: app/connectors/connectors/ftp/ftp-connector/pom.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  <parent>
+    <groupId>io.syndesis.connector</groupId>
+    <artifactId>connector-ftp-parent</artifactId>
+    <version>1.3-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>connector-ftp</artifactId>
+  <name>Connectors :: FTP Connector</name>
```
Comment (lburgazzoli) [inline app/connectors/connectors/ftp/ftp-connector/pom.xml]: Connectors --> Connector (singular)

File: app/connectors/connectors/ftp/sftp-connector/pom.xml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  <parent>
+    <groupId>io.syndesis.connector</groupId>
+    <artifactId>connector-ftp-parent</artifactId>
+    <version>1.3-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>connector-sftp</artifactId>
+  <name>Connectors :: FTP Connector</name>
```
Comment (lburgazzoli) [inline app/connectors/connectors/ftp/sftp-connector/pom.xml]: Connectors --> Connector (singular)
FTP Connector --> SFTP Connector

**Comment by lburgazzoli** [comment]
@sunilmeddr it loggs good to me, some notes:

- I'd like the code to conform with syndesis code style so 4 spaces instead of tabs
- if possible we should add some minimal test cases using [connector-test](https://github.com/syndesisio/syndesis/tree/master/app/connectors/connector-test), for an example see [SqlConnectorTest](https://github.com/syndesisio/syndesis/blob/master/app/connectors/connectors/sql/src/test/java/io/syndesis/connector/sql/SqlConnectorTest.java)

File: app/connectors/connectors/ftp/pom.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    <parent>
+        <groupId>io.syndesis.connector</groupId>
+        <artifactId>connector-parent</artifactId>
+        <version>1.3-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>connector-ftp-parent</artifactId>
+    <name>Connectors :: FTP :: Parent</name>
```
Comment (lburgazzoli) [inline app/connectors/connectors/ftp/pom.xml]: Connectors --> Connector (sorry, I've missed it before)

**Comment by lburgazzoli** [comment]
@sunilmeddr can we switch to spaces also in json files ? once done I'm OK to merge them

**Comment by sunilmeddr** [comment]
@lburgazzoli <https://github.com/lburgazzoli> I have replaced the tab with
4 spaces the json files. Please review

**Comment by lburgazzoli** [comment]
@sunilmeddr thx for the PR

**Pull request merged**